app/helpers/mustache_helper.rb

```diff
@@ -162,6 +162,25 @@ def page_config
     }
   end
 
+  def channel_data
```

Comment:
Assignment Branch Condition size for channel_data is too high. [17.03/15]<br>Method has too many lines. [15/10]<br>Perceived complexity for channel_data is too high. [8/7]